tests/test_Collection.py

```diff
@@ -490,23 +490,19 @@ def test_GetSweepArray_multiprocessing_error():
         assert isinstance(result,ndarray), type_erMsg
         assert allclose(result,mockResult), errMsg  #check if the field results are the same as the mock results in the array
 
-def test_displaySystem_direct_suppress():
-    from magpylib._lib.classes.magnets import Box
-    b = Box([1,1,1],[1,1,1],pos=(5,5,5))
-
-    a = Collection()
+def test_displaySystem_box():
```

Comment:
Rename this to `test_displaySystem` and add a docstring explaining what is expected of this test (what is the behavior being tested?)